browser-extension.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file doesn't need adding to the master branch, it has a special purpose in the FB environment and belongs in the particular branch where it originated.